css/style.css

```diff
@@ -0,0 +1,360 @@
+/* Style.css */
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+body {
+  font-family: helvetica;
+  font-size: 16px;
+}
+
+.stroke {
+  border: 3px solid #fff;
+  background: rgba(0, 0, 0, 0.75);
+  padding: 15px;
+  box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.5);
+}
+
+h1 {
+  font-size:50px;
+  font-family: helvetica;
+  font-weight: bold;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+.align-left {
+  float: left;
+}
+
+.align-right {
+  float: right;
+}
+
+#container {
+  width: 100%;
+  margin: auto;
+  height: 500px;
+}
+
+.hero {
+  text-align: center;
+  background-size: cover;
+  height: 100vh;
+  background: url(../images/art-main.png) no-repeat center top;
+    background-size: cover;
+    background-attachment: fixed;
+    background-position: center;
+    filter: grayscale(100%);
+    -webkit-filter: grayscale(100%); filter: grayscale(100%);
+}
+
+.navigation {
+  float: right;
+  padding-top: 20px;
+  padding-right: 20px;
+}
+
+.grey-background {
+  background-color: #999;
+  overflow: hidden;
+  width: 100%
+}
+
+.navigation ul li {
+  display: inline;
+  list-style-type: none;
+}
+
+.navigation a:link, .navigation a:active, .navigation a:visited {
+  color: #fff;
+  text-decoration: none;
+  font-family: helvetica;
+  font-size: 20px;
+  border: 2px solid white;
+  padding: 10px;
+  margin: 20px;
+}
+
+a:hover {
+  opacity: .7;
+}
+
+.logo {
+  max-width: 100px;
+  float: left;
+  padding: 20px;
+}
+
+.header {
+  padding-top: 250px;
+  color: #fff;
+}
+
+.cta a:link, .cta a:active, .cta a:visited {
```

Comment:
I'm pretty sure if you just select `a` instead of specifying, you'll encompass all of these states.